### ISSUE TITLE
Unknown Error

### ISSUE THREAD

**Issue by 400HPMustang** [thomasloven/hass-browser_mod#339]
Home Assistant Core 2022.6.7
Home Assistant Supervisor 2022.06.2
Home Assistant OS 8.2

**What I am doing:**

I don't know...the error showed up in my logs

**What I expected to happen:**
I expect no error in the logs

**What happened instead:**
I have an error

**Minimal** steps to reproduce:
I have no idea. 

```yaml
# The least amount of code or steps possible to reproduce my error

# Put your code/steps here
```

**Error messages from the browser console:**

```
Logger: homeassistant.components.websocket_api.http.connection
Source: helpers/entity.py:694
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 9:00:07 AM (1 occurrences)
Last logged: 9:00:07 AM

[140297649504752] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/connection.py", line 107, in async_handle
    handler(self.hass, self, schema(msg))
  File "/config/custom_components/browser_mod/connection.py", line 44, in handle_update
    devices[deviceID].update(msg.get("data", None))
  File "/config/custom_components/browser_mod/connection.py", line 108, in update
    self.screen.data = data.get("screen")
  File "/config/custom_components/browser_mod/helpers.py", line 106, in data
    self.updated()
  File "/config/custom_components/browser_mod/light.py", line 28, in updated
    self.schedule_update_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 694, in schedule_update_ha_state
    self.hass.add_job(self.async_update_ha_state(force_refresh))
AttributeError: 'NoneType' object has no attribute 'add_job'

```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by Sergiodiaz53** [comment]
I'm having the same error loading an ui-minimalist control panel, specifically in the mini-graph-card

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
